### PR TITLE
Add CVE-2022-28368 for Dompdf

### PR DESCRIPTION
**Pull request by klausi**
Creating this manually kind of sucks.

Can we somehow import all Github security advisories like https://github.com/advisories/GHSA-x752-qjv4-c4hc for PHP composer projects automatically? https://github.com/advisories?query=type%3Areviewed+ecosystem%3Acomposer

### CODE REVIEW

File: dompdf/dompdf/CVE-2022-28368.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     Remote code injection via remote fonts
+link:      https://github.com/advisories/GHSA-x752-qjv4-c4hc
```
[the remark this answers is not in the document]
Comment (klausi) [reply dompdf/dompdf/CVE-2022-28368.yaml:2]: I would rather link to the Github advisory, they do a good job linking to all further resources like https://github.com/dompdf/dompdf/issues/2598

[the remark this answers is not in the document]
Comment (stof) [reply dompdf/dompdf/CVE-2022-28368.yaml:2]: but the link itself provide no details about the vulnerability if you don't go to other links

**Comment by klausi** [comment]
Opened #626 for automating this process.

**Review by stof — APPROVED** [review]
OK, let's merge this as is.

**Comment by klausi** [comment]
Thank you, appreciated!

**Pull request merged**